src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -158,8 +160,8 @@ public AdyenPaymentPluginApi(final AdyenConfigurationHandler adyenConfigurationH
         this.adyenHppConfigurationHandler = adyenHppConfigurationHandler;
         this.dao = dao;
 
-        final KillbillAdyenNotificationHandler adyenNotificationHandler = new KillbillAdyenNotificationHandler(killbillApi, dao, clock);
-        this.adyenNotificationService = new AdyenNotificationService(ImmutableList.<AdyenNotificationHandler>of(adyenNotificationHandler));
+        final AdyenNotificationHandler adyenNotificationHandler = new KillbillAdyenNotificationHandler(killbillApi, dao, clock);
```

Comment:
Can you keep `<AdyenNotificationHandler>`? Type inference is broken on old 1.6 jdk versions.